### ISSUE TITLE
Match using `iOS Developer` certificate instead `Apple Distribution` in Github Actions CI

### ISSUE THREAD

**Issue by ggirotto** [fastlane/fastlane#19441]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description
I'm using Github Actions to automatically deploy an iOS app. When using `match` to download provisions and certificates, it fails to archive the application with the error:

`error: No signing certificate "iOS Distribution" found: No "iOS Distribution" signing certificate matching team ID "xxxxx" with a private key was found. (in target 'xxxx' from project 'xxxx')`

I'm struggling to figure our what's happening, since when I run in my local machine everything work as expected. Both, my machine and Github Actions are running in macOS.

The only hypothesis that I could find to this work in my local machine and don't in Github Actions are:

- 1. Issues related to Github Actions Keychain access for storing or retrieving the certificates.
- 2. Based on the error, fastlane is searching for an iOS Distribution certificate, although we are using `Apple Distribution` certificates to sign the app for release.

I'm not sure what's happening, but I'm starting to think that the second hypothesis may be the issue based on the error description.

##### Command executed
My lane that setup certificates and probisions:

```ruby

default_platform(:ios)

platform :ios do

  ########################--------------------------------########################
  ########################--------- SETUP LANES ----------########################
  ########################--------------------------------########################

  # Environment Setup
  ## Setup local environment configuration. If running on a CI machine we need to create a Keychain to store the 
  ## distribution certificates
  private_lane :setup_environment do
    if is_ci
      setup_ci
    end
  end

  # Provisions and Certificates Setup
  ## Download and setup all necessary provisions and certificates (Dev & Dist)
  desc "Setup Development & Distribution profiles and certificates in the local machine"
  lane :setup_provisions do
    setup_environment

    # Install Distribution Provisions and Certificates
    match(
      app_identifier: "<APP_ID>",
      type: "appstore",
      readonly: true,
      verbose: true,
    )

    # Install Development Provisions and Certificates
    match(
      app_identifier: "<APP_ID>",
      type: "development",
      readonly: true,
      verbose: true,
    )
  end
end

```

##### Complete output when running fastlane, including the stack trace and command used

<details>
  <pre> cd ios && fastlane release --verbose
  shell: /bin/bash -e ***0***
  env:
    flutter_version: 2.2.3
    java_version: 12.x
    flutter_channel: stable
    JAVA_HOME_12.0.2_x64: /Users/runner/hostedtoolcache/jdk/12.0.2/x64
    JAVA_HOME: /Users/runner/hostedtoolcache/jdk/12.0.2/x64
    JAVA_HOME_12_0_2_X64: /Users/runner/hostedtoolcache/jdk/12.0.2/x64
    FLUTTER_ROOT: /Users/runner/hostedtoolcache/flutter/2.2.3-stable/x64
    PUB_CACHE: /Users/runner/hostedtoolcache/flutter/2.2.3-stable/x64/.pub-cache
    APP_STORE_API_KEY: ***
    APP_STORE_API_KEY_ID: ***
    APP_STORE_API_KEY_ISSUER_ID: ***
    APPLE_ID: ***
    ITC_TEAM_ID: ***
    TEAM_ID: ***
    KEYCHAIN_NAME: ***
    KEYCHAIN_PASSWORD: ***
    PROVISIONS_CERTIFICATES_REPO: ***
    MATCH_PASSWORD: ***
    FIREBASE_APP_ID: ***
[19:31:15]: fastlane detected a Gemfile in the current directory
[19:31:15]: However, it seems like you didn't use `bundle exec`
[19:31:15]: To launch fastlane faster, please use
[19:31:15]: 
[19:31:15]: $ bundle exec fastlane release --verbose
[19:31:15]: 
[19:31:15]: Get started using a Gemfile for fastlane https://docs.fastlane.tools/getting-started/ios/setup/#use-a-gemfile
DEBUG [2021-10-05 19:31:17.35]: Checking if there are any plugins that should be loaded...
Successfully loaded Appfile at path '/Users/runner/work/studo-tests/studo-tests/flutter/ios/fastlane/Appfile'
- app_identifier: 'com.olmps.ticker'
- apple_id: '***'
- itc_team_id: '***'
- team_id: '***'
-------
INFO [2021-10-05 19:31:17.66]: Sending anonymous analytics information
INFO [2021-10-05 19:31:17.66]: Learn more at https://docs.fastlane.tools/#metrics
INFO [2021-10-05 19:31:17.66]: No personal or sensitive data is sent.
INFO [2021-10-05 19:31:17.66]: You can disable this by adding `opt_out_usage` at the top of your Fastfile
INFO [2021-10-05 19:31:17.75]: ------------------------------
INFO [2021-10-05 19:31:17.75]: --- Step: default_platform ---
INFO [2021-10-05 19:31:17.75]: ------------------------------
INFO [2021-10-05 19:31:17.75]: Driving the lane 'ios release' 🚀
INFO [2021-10-05 19:31:17.75]: ---------------------------------------
INFO [2021-10-05 19:31:17.75]: --- Step: app_store_connect_api_key ---
INFO [2021-10-05 19:31:17.75]: ---------------------------------------
INFO [2021-10-05 19:31:17.76]: -------------------------------------------------
INFO [2021-10-05 19:31:17.76]: --- Step: Switch to ios setup_provisions lane ---
INFO [2021-10-05 19:31:17.76]: -------------------------------------------------
INFO [2021-10-05 19:31:17.76]: Cruising over to lane 'ios setup_provisions' 🚖
INFO [2021-10-05 19:31:17.76]: ----------------------
INFO [2021-10-05 19:31:17.76]: --- Step: setup_ci ---
INFO [2021-10-05 19:31:17.76]: ----------------------
INFO [2021-10-05 19:31:17.76]: Creating temporary keychain: "fastlane_tmp_keychain".
INFO [2021-10-05 19:31:17.89]: $ security list-keychains -d user
INFO [2021-10-05 19:31:17.90]: ▸ "/Users/runner/Library/Keychains/fastlane_tmp_keychain-db"
WARN [2021-10-05 19:31:17.90]: Found keychain '/Users/runner/Library/Keychains/fastlane_tmp_keychain-db' in list-keychains, adding to search list skipped
INFO [2021-10-05 19:31:17.90]: Enabling match readonly mode.
INFO [2021-10-05 19:31:18.36]: -------------------
INFO [2021-10-05 19:31:18.36]: --- Step: match ---
INFO [2021-10-05 19:31:18.36]: -------------------
INFO [2021-10-05 19:31:18.36]: Successfully loaded '/Users/runner/work/studo-tests/studo-tests/flutter/ios/fastlane/Matchfile' 📄

+--------------+---------------------------------------------------+
|           Detected Values from './fastlane/Matchfile'            |
+--------------+---------------------------------------------------+
| git_url      | *** |
| storage_mode | git                                               |
+--------------+---------------------------------------------------+

+--------------------------------+---------------------------------------------------+
|                             Summary for match 2.195.0                              |
+--------------------------------+---------------------------------------------------+
| app_identifier                 | ["com.olmps.ticker"]                              |
| type                           | appstore                                          |
| readonly                       | true                                              |
| verbose                        | true                                              |
| api_key                        | ********                                          |
| generate_apple_certs           | true                                              |
| skip_provisioning_profiles     | false                                             |
| username                       | ***                                |
| team_id                        | ***                                        |
| storage_mode                   | git                                               |
| git_url                        | *** |
| git_branch                     | master                                            |
| shallow_clone                  | false                                             |
| clone_branch_directly          | false                                             |
| keychain_name                  | fastlane_tmp_keychain                             |
| force                          | false                                             |
| force_for_new_devices          | false                                             |
| include_all_certificates       | false                                             |
| force_for_new_certificates     | false                                             |
| skip_confirmation              | false                                             |
| skip_docs                      | false                                             |
| platform                       | ios                                               |
| derive_catalyst_app_identifier | false                                             |
| fail_on_name_taken             | false                                             |
| skip_certificate_matching      | false                                             |
| skip_set_partition_list        | false                                             |
+--------------------------------+---------------------------------------------------+

INFO [2021-10-05 19:31:18.37]: Cloning remote git repo...
INFO [2021-10-05 19:31:18.37]: If cloning the repo takes too long, you can use the `clone_branch_directly` option in match.
INFO [2021-10-05 19:31:18.37]: $ git clone *** /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/d20211005-8163-buu7sa
INFO [2021-10-05 19:31:18.42]: ▸ Cloning into '/var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/d20211005-8163-buu7sa'...
INFO [2021-10-05 19:31:18.82]: ▸ Warning: Permanently added the RSA host key for IP address '192.30.255.112' to the list of known hosts.
INFO [2021-10-05 19:31:19.68]: ▸ remote: Enumerating objects: 266, done.        
INFO [2021-10-05 19:31:19.68]: ▸ remote: Counting objects: 100% (266/266), done.        
INFO [2021-10-05 19:31:19.75]: ▸ remote: Compressing objects: 100% (228/228), done.        
INFO [2021-10-05 19:31:20.06]: ▸ remote: Total 266 (delta 27), reused 249 (delta 10), pack-reused 0        
INFO [2021-10-05 19:31:20.06]: ▸ Receiving objects: 100% (266/266), 393.49 KiB | 1.21 MiB/s, done.
INFO [2021-10-05 19:31:20.06]: ▸ Resolving deltas: 100% (27/27), done.
INFO [2021-10-05 19:31:20.09]: $ git --no-pager branch --list origin/master --no-color -r
INFO [2021-10-05 19:31:20.10]: ▸   origin/master
INFO [2021-10-05 19:31:20.10]: Checking out branch master...
INFO [2021-10-05 19:31:20.10]: $ git checkout master
INFO [2021-10-05 19:31:20.12]: ▸ Already on 'master'
INFO [2021-10-05 19:31:20.12]: ▸ Your branch is up to date with 'origin/master'.
INFO [2021-10-05 19:31:20.12]: 🔓  Decrypted 'YL5SU74925.cer'
INFO [2021-10-05 19:31:20.12]: 🔓  Decrypted '8M9LXYSTR2.cer'
INFO [2021-10-05 19:31:20.12]: 🔓  Decrypted '8M9LXYSTR2.p12'
INFO [2021-10-05 19:31:20.12]: 🔓  Decrypted 'YL5SU74925.p12'
INFO [2021-10-05 19:31:20.12]: 🔓  Decrypted 'ZP7XR68H33.cer'
INFO [2021-10-05 19:31:20.12]: 🔓  Decrypted 'GTNHN3TB3R.cer'
INFO [2021-10-05 19:31:20.12]: 🔓  Decrypted 'GTNHN3TB3R.p12'
INFO [2021-10-05 19:31:20.12]: 🔓  Decrypted 'ZP7XR68H33.p12'
INFO [2021-10-05 19:31:20.12]: 🔓  Decrypted 'AppStore_com.olmps.ticker.mobileprovision'
INFO [2021-10-05 19:31:20.12]: 🔓  Decrypted 'Development_com.olmps.ticker.dev.mobileprovision'
INFO [2021-10-05 19:31:20.12]: 🔓  Successfully decrypted certificates repo
DEBUG [2021-10-05 19:31:20.12]: Your certificate 'GTNHN3TB3R.cer' is valid
INFO [2021-10-05 19:31:20.12]: Installing certificate...
INFO [2021-10-05 19:31:20.12]: $ security list-keychains -d user
INFO [2021-10-05 19:31:20.13]: ▸     "/Users/runner/Library/Keychains/fastlane_tmp_keychain-db"
INFO [2021-10-05 19:31:20.13]: $ security find-certificate -a -c 'Apple Worldwide Developer Relations Certification Authority' /Users/runner/Library/Keychains/fastlane_tmp_keychain-db
INFO [2021-10-05 19:31:20.16]: $ security list-keychains -d user
INFO [2021-10-05 19:31:20.18]: ▸     "/Users/runner/Library/Keychains/fastlane_tmp_keychain-db"
DEBUG [2021-10-05 19:31:20.18]: Installing WWDR Cert: curl -f -o /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/AppleWWDRCA.cer20211005-8163-14ty0zz https://developer.apple.com/certificationauthority/AppleWWDRCA.cer && security import /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/AppleWWDRCA.cer20211005-8163-14ty0zz -k /Users/runner/Library/Keychains/fastlane_tmp_keychain-db
INFO [2021-10-05 19:31:20.41]: ▸ 1 certificate imported.
INFO [2021-10-05 19:31:20.41]: ▸ 100  1062  100  1062    0     0  16079      0 --:--:-- --:--:-- --:--:-- 19309
INFO [2021-10-05 19:31:20.41]: $ security list-keychains -d user
INFO [2021-10-05 19:31:20.43]: ▸     "/Users/runner/Library/Keychains/fastlane_tmp_keychain-db"
DEBUG [2021-10-05 19:31:20.43]: Installing WWDR Cert: curl -f -o /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/AppleWWDRCAG3.cer20211005-8163-1dwn8r7 https://www.apple.com/certificateauthority/AppleWWDRCAG3.cer && security import /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/AppleWWDRCAG3.cer20211005-8163-1dwn8r7 -k /Users/runner/Library/Keychains/fastlane_tmp_keychain-db
INFO [2021-10-05 19:31:20.56]: ▸ 1 certificate imported.
INFO [2021-10-05 19:31:20.56]: ▸ 100  1109  100  1109    0     0  13903      0 --:--:-- --:--:-- --:--:-- 14986
ERROR [2021-10-05 19:31:20.59]: There are no local code signing identities found.
You can run `security find-identity -v -p codesigning fastlane_tmp_keychain` to get this output.
This Stack Overflow thread has more information: https://stackoverflow.com/q/35390072/774.
(Check in Keychain Access for an expired WWDR certificate: https://stackoverflow.com/a/35409835/774 has more info.)
INFO [2021-10-05 19:31:20.65]: $ security import /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/d20211005-8163-buu7sa/certs/distribution/GTNHN3TB3R.cer -k '/Users/runner/Library/Keychains/fastlane_tmp_keychain-db' -P ******** -T /usr/bin/codesign -T /usr/bin/security -T /usr/bin/productbuild
INFO [2021-10-05 19:31:20.71]: ▸ 1 certificate imported.
INFO [2021-10-05 19:31:20.72]: Setting key partition list... (this can take a minute if there are a lot of keys installed)
INFO [2021-10-05 19:31:20.72]: $ security set-key-partition-list -S apple-tool:,apple:,codesign: -s -k ******** /Users/runner/Library/Keychains/fastlane_tmp_keychain-db 1> /dev/null
ERROR [2021-10-05 19:31:20.75]: security: SecItemCopyMatching: The specified item could not be found in the keychain.
INFO [2021-10-05 19:31:20.75]: $ security import /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/d20211005-8163-buu7sa/certs/distribution/ZP7XR68H33.p12 -k '/Users/runner/Library/Keychains/fastlane_tmp_keychain-db' -P ******** -T /usr/bin/codesign -T /usr/bin/security -T /usr/bin/productbuild
INFO [2021-10-05 19:31:20.84]: ▸ 1 identity imported.
INFO [2021-10-05 19:31:20.85]: Setting key partition list... (this can take a minute if there are a lot of keys installed)
INFO [2021-10-05 19:31:20.85]: $ security set-key-partition-list -S apple-tool:,apple:,codesign: -s -k ******** /Users/runner/Library/Keychains/fastlane_tmp_keychain-db 1> /dev/null

+-------------------+--------------------------------------------------------+
|                           Installed Certificate                            |
+-------------------+--------------------------------------------------------+
| User ID           | 9TW867US97                                             |
| Common Name       | Apple Distribution: Oplab Tecnologia LTDA (9TW867US97) |
| Organisation Unit | 9TW867US97                                             |
| Organisation      | Oplab Tecnologia LTDA                                  |
| Country           | US                                                     |
| Start Datetime    | 2021-09-21 13:57:45 UTC                                |
| End Datetime      | 2022-09-21 13:57:44 UTC                                |
+-------------------+--------------------------------------------------------+

INFO [2021-10-05 19:31:20.90]: Installing provisioning profile...
WARN [2021-10-05 19:31:21.09]: Setting environment variable 'sigh_com.olmps.ticker_appstore' to '2af27c3e-8373-479c-bafc-0da6189ec5cb'
WARN [2021-10-05 19:31:21.09]: Setting environment variable 'sigh_com.olmps.ticker_appstore_team-id' to '9TW867US97'
WARN [2021-10-05 19:31:21.09]: Setting environment variable 'sigh_com.olmps.ticker_appstore_profile-name' to 'Studo Distribution'
WARN [2021-10-05 19:31:21.09]: Setting environment variable 'sigh_com.olmps.ticker_appstore_profile-path' to '/Users/runner/Library/MobileDevice/Provisioning Profiles/2af27c3e-8373-479c-bafc-0da6189ec5cb.mobileprovision'

+---------------------+---------------------------------------------+---------------------------------------------------------------------------------------------------------------+
|                                                                          Installed Provisioning Profile                                                                           |
+---------------------+---------------------------------------------+---------------------------------------------------------------------------------------------------------------+
| Parameter           | Environment Variable                        | Value                                                                                                         |
+---------------------+---------------------------------------------+---------------------------------------------------------------------------------------------------------------+
| App Identifier      |                                             | com.olmps.ticker                                                                                              |
| Type                |                                             | appstore                                                                                                      |
| Platform            |                                             | ios                                                                                                           |
| Profile UUID        | sigh_com.olmps.ticker_appstore              | 2af27c3e-8373-479c-bafc-0da6189ec5cb                                                                          |
| Profile Name        | sigh_com.olmps.ticker_appstore_profile-name | Studo Distribution                                                                                            |
| Profile Path        | sigh_com.olmps.ticker_appstore_profile-path | /Users/runner/Library/MobileDevice/Provisioning Profiles/2af27c3e-8373-479c-bafc-0da6189ec5cb.mobileprovision |
| Development Team ID | sigh_com.olmps.ticker_appstore_team-id      | 9TW867US97                                                                                                    |
+---------------------+---------------------------------------------+---------------------------------------------------------------------------------------------------------------+

INFO [2021-10-05 19:31:21.09]: All required keys, certificates and provisioning profiles are installed 🙌
INFO [2021-10-05 19:31:21.10]: Setting Provisioning Profile type to 'app-store'
INFO [2021-10-05 19:31:21.11]: -------------------
INFO [2021-10-05 19:31:21.11]: --- Step: match ---
INFO [2021-10-05 19:31:21.11]: -------------------
INFO [2021-10-05 19:31:21.11]: Successfully loaded '/Users/runner/work/studo-tests/studo-tests/flutter/ios/fastlane/Matchfile' 📄

+--------------+---------------------------------------------------+
|           Detected Values from './fastlane/Matchfile'            |
+--------------+---------------------------------------------------+
| git_url      | *** |
| storage_mode | git                                               |
+--------------+---------------------------------------------------+

+--------------------------------+---------------------------------------------------+
|                             Summary for match 2.195.0                              |
+--------------------------------+---------------------------------------------------+
| app_identifier                 | ["com.olmps.ticker.dev"]                          |
| type                           | development                                       |
| readonly                       | true                                              |
| verbose                        | true                                              |
| api_key                        | ********                                          |
| generate_apple_certs           | true                                              |
| skip_provisioning_profiles     | false                                             |
| username                       | ***                                |
| team_id                        | ***                                        |
| storage_mode                   | git                                               |
| git_url                        | *** |
| git_branch                     | master                                            |
| shallow_clone                  | false                                             |
| clone_branch_directly          | false                                             |
| keychain_name                  | fastlane_tmp_keychain                             |
| force                          | false                                             |
| force_for_new_devices          | false                                             |
| include_all_certificates       | false                                             |
| force_for_new_certificates     | false                                             |
| skip_confirmation              | false                                             |
| skip_docs                      | false                                             |
| platform                       | ios                                               |
| derive_catalyst_app_identifier | false                                             |
| fail_on_name_taken             | false                                             |
| skip_certificate_matching      | false                                             |
| skip_set_partition_list        | false                                             |
+--------------------------------+---------------------------------------------------+

INFO [2021-10-05 19:31:21.12]: Cloning remote git repo...
INFO [2021-10-05 19:31:21.12]: If cloning the repo takes too long, you can use the `clone_branch_directly` option in match.
INFO [2021-10-05 19:31:21.12]: $ git clone *** /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/d20211005-8163-1chuj3n
INFO [2021-10-05 19:31:21.13]: ▸ Cloning into '/var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/d20211005-8163-1chuj3n'...
INFO [2021-10-05 19:31:22.28]: ▸ remote: Enumerating objects: 266, done.        
INFO [2021-10-05 19:31:22.28]: ▸ remote: Counting objects: 100% (266/266), done.        
INFO [2021-10-05 19:31:22.34]: ▸ remote: Compressing objects: 100% (228/228), done.        
INFO [2021-10-05 19:31:23.31]: ▸ remote: Total 266 (delta 27), reused 249 (delta 10), pack-reused 0        
INFO [2021-10-05 19:31:23.31]: ▸ Receiving objects: 100% (266/266), 393.49 KiB | 437.00 KiB/s, done.
INFO [2021-10-05 19:31:23.31]: ▸ Resolving deltas: 100% (27/27), done.
INFO [2021-10-05 19:31:23.34]: $ git --no-pager branch --list origin/master --no-color -r
INFO [2021-10-05 19:31:23.35]: ▸   origin/master
INFO [2021-10-05 19:31:23.35]: Checking out branch master...
INFO [2021-10-05 19:31:23.35]: $ git checkout master
INFO [2021-10-05 19:31:23.36]: ▸ Already on 'master'
INFO [2021-10-05 19:31:23.36]: ▸ Your branch is up to date with 'origin/master'.
INFO [2021-10-05 19:31:23.37]: 🔓  Decrypted 'YL5SU74925.cer'
INFO [2021-10-05 19:31:23.37]: 🔓  Decrypted '8M9LXYSTR2.cer'
INFO [2021-10-05 19:31:23.37]: 🔓  Decrypted '8M9LXYSTR2.p12'
INFO [2021-10-05 19:31:23.37]: 🔓  Decrypted 'YL5SU74925.p12'
INFO [2021-10-05 19:31:23.37]: 🔓  Decrypted 'ZP7XR68H33.cer'
INFO [2021-10-05 19:31:23.37]: 🔓  Decrypted 'GTNHN3TB3R.cer'
INFO [2021-10-05 19:31:23.37]: 🔓  Decrypted 'GTNHN3TB3R.p12'
INFO [2021-10-05 19:31:23.37]: 🔓  Decrypted 'ZP7XR68H33.p12'
INFO [2021-10-05 19:31:23.37]: 🔓  Decrypted 'AppStore_com.olmps.ticker.mobileprovision'
INFO [2021-10-05 19:31:23.37]: 🔓  Decrypted 'Development_com.olmps.ticker.dev.mobileprovision'
INFO [2021-10-05 19:31:23.37]: 🔓  Successfully decrypted certificates repo
DEBUG [2021-10-05 19:31:23.37]: Your certificate '8M9LXYSTR2.cer' is valid
INFO [2021-10-05 19:31:23.37]: Installing certificate...
INFO [2021-10-05 19:31:23.37]: $ security list-keychains -d user
INFO [2021-10-05 19:31:23.38]: ▸     "/Users/runner/Library/Keychains/fastlane_tmp_keychain-db"
INFO [2021-10-05 19:31:23.38]: $ security find-certificate -a -c 'Apple Worldwide Developer Relations Certification Authority' /Users/runner/Library/Keychains/fastlane_tmp_keychain-db
INFO [2021-10-05 19:31:23.41]: ▸ keychain: "/Users/runner/Library/Keychains/fastlane_tmp_keychain-db"
INFO [2021-10-05 19:31:23.41]: ▸ version: 512
INFO [2021-10-05 19:31:23.41]: ▸ class: 0x80001000 
INFO [2021-10-05 19:31:23.41]: ▸ attributes:
INFO [2021-10-05 19:31:23.41]: ▸     "alis"<blob>="Apple Worldwide Developer Relations Certification Authority"
INFO [2021-10-05 19:31:23.41]: ▸     "cenc"<uint32>=0x00000003 
INFO [2021-10-05 19:31:23.41]: ▸     "ctyp"<uint32>=0x00000001 
INFO [2021-10-05 19:31:23.41]: ▸     "hpky"<blob>=0x88271709A9B618608BECEBBAF64759C55254A3B7  "\210'\027\011\251\266\030`\213\354\353\272\366GY\305RT\243\267"
INFO [2021-10-05 19:31:23.41]: ▸     "issu"<blob>=0x3062310B300906035504061302555331133011060355040A130A4150504C4520494E432E31263024060355040B131D4150504C452043455254494649434154494F4E20415554484F52495459311630140603550403130D4150504C4520524F4F54204341  "0b1\0130\011\006\003U\004\006\023\002US1\0230\021\006\003U\004\012\023\012APPLE INC.1&0$\006\003U\004\013\023\035APPLE CERTIFICATION AUTHORITY1\0260\024\006\003U\004\003\023\015APPLE ROOT CA"
INFO [2021-10-05 19:31:23.41]: ▸     "labl"<blob>="Apple Worldwide Developer Relations Certification Authority"
INFO [2021-10-05 19:31:23.41]: ▸     "skid"<blob>=0x88271709A9B618608BECEBBAF64759C55254A3B7  "\210'\027\011\251\266\030`\213\354\353\272\366GY\305RT\243\267"
INFO [2021-10-05 19:31:23.41]: ▸     "snbr"<blob>=0x01DEBCC4396DA010  "\001\336\274\3049m\240\020"
INFO [2021-10-05 19:31:23.41]: ▸     "subj"<blob>=0x308196310B300906035504061302555331133011060355040A0C0A4170706C6520496E632E312C302A060355040B0C234170706C6520576F726C647769646520446576656C6F7065722052656C6174696F6E733144304206035504030C3B4170706C6520576F726C647769646520446576656C6F7065722052656C6174696F6E732043657274696669636174696F6E20417574686F72697479  "0\201\2261\0130\011\006\003U\004\006\023\002US1\0230\021\006\003U\004\012\014\012Apple Inc.1,0*\006\003U\004\013\014#Apple Worldwide Developer Relations1D0B\006\003U\004\003\014;Apple Worldwide Developer Relations Certification Authority"
INFO [2021-10-05 19:31:23.41]: ▸ keychain: "/Users/runner/Library/Keychains/fastlane_tmp_keychain-db"
INFO [2021-10-05 19:31:23.41]: ▸ version: 512
INFO [2021-10-05 19:31:23.41]: ▸ class: 0x80001000 
INFO [2021-10-05 19:31:23.41]: ▸ attributes:
INFO [2021-10-05 19:31:23.41]: ▸     "alis"<blob>="Apple Worldwide Developer Relations Certification Authority"
INFO [2021-10-05 19:31:23.41]: ▸     "cenc"<uint32>=0x00000003 
INFO [2021-10-05 19:31:23.41]: ▸     "ctyp"<uint32>=0x00000001 
INFO [2021-10-05 19:31:23.41]: ▸     "hpky"<blob>=0x09FEC01590F9AF640A9212B92628630C97ECA7B2  "\011\376\300\025\220\371\257d\012\222\022\271&(c\014\227\354\247\262"
INFO [2021-10-05 19:31:23.41]: ▸     "issu"<blob>=0x3062310B300906035504061302555331133011060355040A130A4150504C4520494E432E31263024060355040B131D4150504C452043455254494649434154494F4E20415554484F52495459311630140603550403130D4150504C4520524F4F54204341  "0b1\0130\011\006\003U\004\006\023\002US1\0230\021\006\003U\004\012\023\012APPLE INC.1&0$\006\003U\004\013\023\035APPLE CERTIFICATION AUTHORITY1\0260\024\006\003U\004\003\023\015APPLE ROOT CA"
INFO [2021-10-05 19:31:23.41]: ▸     "labl"<blob>="Apple Worldwide Developer Relations Certification Authority"
INFO [2021-10-05 19:31:23.41]: ▸     "skid"<blob>=0x09FEC01590F9AF640A9212B92628630C97ECA7B2  "\011\376\300\025\220\371\257d\012\222\022\271&(c\014\227\354\247\262"
INFO [2021-10-05 19:31:23.41]: ▸     "snbr"<blob>=0x7CAF690A25B739FE7B9B447AC178C5EE  "|\257i\012%\2679\376***\233Dz\301x\305\356"
INFO [2021-10-05 19:31:23.41]: ▸     "subj"<blob>=0x30753144304206035504030C3B4170706C6520576F726C647769646520446576656C6F7065722052656C6174696F6E732043657274696669636174696F6E20417574686F72697479310B3009060355040B0C02473331133011060355040A0C0A4170706C6520496E632E310B3009060355040613025553  "0u1D0B\006\003U\004\003\014;Apple Worldwide Developer Relations Certification Authority1\0130\011\006\003U\004\013\014\002G31\0230\021\006\003U\004\012\014\012Apple Inc.1\0130\011\006\003U\004\006\023\002US"
INFO [2021-10-05 19:31:23.60]: $ security import /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/d20211005-8163-1chuj3n/certs/development/8M9LXYSTR2.cer -k '/Users/runner/Library/Keychains/fastlane_tmp_keychain-db' -P ******** -T /usr/bin/codesign -T /usr/bin/security -T /usr/bin/productbuild
INFO [2021-10-05 19:31:23.66]: ▸ 1 certificate imported.
INFO [2021-10-05 19:31:23.67]: Setting key partition list... (this can take a minute if there are a lot of keys installed)
INFO [2021-10-05 19:31:23.67]: $ security set-key-partition-list -S apple-tool:,apple:,codesign: -s -k ******** /Users/runner/Library/Keychains/fastlane_tmp_keychain-db 1> /dev/null
INFO [2021-10-05 19:31:23.72]: $ security import /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/d20211005-8163-1chuj3n/certs/development/YL5SU74925.p12 -k '/Users/runner/Library/Keychains/fastlane_tmp_keychain-db' -P ******** -T /usr/bin/codesign -T /usr/bin/security -T /usr/bin/productbuild
INFO [2021-10-05 19:31:23.80]: ▸ 1 identity imported.
INFO [2021-10-05 19:31:23.81]: Setting key partition list... (this can take a minute if there are a lot of keys installed)
INFO [2021-10-05 19:31:23.81]: $ security set-key-partition-list -S apple-tool:,apple:,codesign: -s -k ******** /Users/runner/Library/Keychains/fastlane_tmp_keychain-db 1> /dev/null

+-------------------+---------------------------------------------------+
|                         Installed Certificate                         |
+-------------------+---------------------------------------------------+
| User ID           | KEA84Q3ZRH                                        |
| Common Name       | Apple Development: Guilherme Girotto (KP6W2SKUCD) |
| Organisation Unit | 9TW867US97                                        |
| Organisation      | Oplab Tecnologia LTDA                             |
| Country           | BR                                                |
| Start Datetime    | 2021-09-21 14:06:49 UTC                           |
| End Datetime      | 2022-09-21 14:06:48 UTC                           |
+-------------------+---------------------------------------------------+

INFO [2021-10-05 19:31:23.86]: Installing provisioning profile...
WARN [2021-10-05 19:31:24.01]: Setting environment variable 'sigh_com.olmps.ticker.dev_development' to '770e248d-b13f-453f-876c-594e2b4f2ff9'
WARN [2021-10-05 19:31:24.01]: Setting environment variable 'sigh_com.olmps.ticker.dev_development_team-id' to '9TW867US97'
WARN [2021-10-05 19:31:24.01]: Setting environment variable 'sigh_com.olmps.ticker.dev_development_profile-name' to 'Studo Development'
WARN [2021-10-05 19:31:24.01]: Setting environment variable 'sigh_com.olmps.ticker.dev_development_profile-path' to '/Users/runner/Library/MobileDevice/Provisioning Profiles/770e248d-b13f-453f-876c-594e2b4f2ff9.mobileprovision'

+---------------------+----------------------------------------------------+---------------------------------------------------------------------------------------------------------------+
|                                                                              Installed Provisioning Profile                                                                              |
+---------------------+----------------------------------------------------+---------------------------------------------------------------------------------------------------------------+
| Parameter           | Environment Variable                               | Value                                                                                                         |
+---------------------+----------------------------------------------------+---------------------------------------------------------------------------------------------------------------+
| App Identifier      |                                                    | com.olmps.ticker.dev                                                                                          |
| Type                |                                                    | development                                                                                                   |
| Platform            |                                                    | ios                                                                                                           |
| Profile UUID        | sigh_com.olmps.ticker.dev_development              | 770e248d-b13f-453f-876c-594e2b4f2ff9                                                                          |
| Profile Name        | sigh_com.olmps.ticker.dev_development_profile-name | Studo Development                                                                                             |
| Profile Path        | sigh_com.olmps.ticker.dev_development_profile-path | /Users/runner/Library/MobileDevice/Provisioning Profiles/770e248d-b13f-453f-876c-594e2b4f2ff9.mobileprovision |
| Development Team ID | sigh_com.olmps.ticker.dev_development_team-id      | 9TW867US97                                                                                                    |
+---------------------+----------------------------------------------------+---------------------------------------------------------------------------------------------------------------+

INFO [2021-10-05 19:31:24.01]: All required keys, certificates and provisioning profiles are installed 🙌
INFO [2021-10-05 19:31:24.02]: Setting Provisioning Profile type to 'development'
INFO [2021-10-05 19:31:24.02]: Cruising back to lane 'ios release' 🚘
INFO [2021-10-05 19:31:24.02]: ---------------------------
INFO [2021-10-05 19:31:24.02]: --- Step: flutter clean ---
INFO [2021-10-05 19:31:24.02]: ---------------------------
INFO [2021-10-05 19:31:24.02]: $ flutter clean
INFO [2021-10-05 19:31:24.87]: ▸ Changing current working directory to: /Users/runner/work/studo-tests/studo-tests/flutter
INFO [2021-10-05 19:31:36.01]: ▸ Cleaning Xcode workspace...                                        10.9s
INFO [2021-10-05 19:31:36.03]: ▸ Deleting build...                                                   14ms
INFO [2021-10-05 19:31:36.03]: ▸ Deleting .dart_tool...                                               0ms
INFO [2021-10-05 19:31:36.03]: ▸ Deleting .packages...                                                0ms
INFO [2021-10-05 19:31:36.03]: ▸ Deleting Generated.xcconfig...                                       0ms
INFO [2021-10-05 19:31:36.03]: ▸ Deleting flutter_export_environment.sh...                            0ms
INFO [2021-10-05 19:31:36.04]: ▸ Deleting .flutter-plugins-dependencies...                            0ms
INFO [2021-10-05 19:31:36.04]: ▸ Deleting .flutter-plugins...                                         0ms
INFO [2021-10-05 19:31:36.06]: -----------------------------
INFO [2021-10-05 19:31:36.06]: --- Step: flutter pub get ---
INFO [2021-10-05 19:31:36.06]: -----------------------------
INFO [2021-10-05 19:31:36.06]: $ flutter pub get
INFO [2021-10-05 19:31:36.89]: ▸ Changing current working directory to: /Users/runner/work/studo-tests/studo-tests/flutter
INFO [2021-10-05 19:31:39.28]: ▸ Running "flutter pub get" in flutter...                          2,205ms
INFO [2021-10-05 19:31:39.88]: --------------------------------
INFO [2021-10-05 19:31:39.88]: --- Step: get_version_number ---
INFO [2021-10-05 19:31:39.88]: --------------------------------
INFO [2021-10-05 19:31:39.99]: --------------------------------------------
INFO [2021-10-05 19:31:39.99]: --- Step: latest_testflight_build_number ---
INFO [2021-10-05 19:31:39.99]: --------------------------------------------
INFO [2021-10-05 19:31:39.99]: Creating authorization token for App Store Connect API
INFO [2021-10-05 19:31:42.41]: Fetching the latest build number for version 0.1.0
INFO [2021-10-05 19:31:42.56]: Latest upload for version 0.1.0 on ios platform is build: 21
INFO [2021-10-05 19:31:42.56]: ------------------------------------
INFO [2021-10-05 19:31:42.56]: --- Step: increment_build_number ---
INFO [2021-10-05 19:31:42.56]: ------------------------------------
Current version of project Runner is: 
    20

/Users/runner/work/studo-tests/studo-tests/flutter/ios
INFO [2021-10-05 19:31:44.70]: $ cd /Users/runner/work/studo-tests/studo-tests/flutter/ios && agvtool new-version -all 22 && cd -
INFO [2021-10-05 19:31:45.13]: ▸ Setting version of project Runner to:
INFO [2021-10-05 19:31:45.13]: ▸ 22.
INFO [2021-10-05 19:31:45.15]: ▸ Also setting CFBundleVersion key (assuming it exists)
INFO [2021-10-05 19:31:45.17]: ▸ Updating CFBundleVersion in Info.plist(s)...
INFO [2021-10-05 19:31:45.19]: ▸ Updated CFBundleVersion in "Runner.xcodeproj/../Runner/info-plist/dev/Info.plist" to 22
INFO [2021-10-05 19:31:45.20]: ▸ Updated CFBundleVersion in "Runner.xcodeproj/../Runner/info-plist/prod/Info.plist" to 22
INFO [2021-10-05 19:31:45.20]: ▸ /Users/runner/work/studo-tests/studo-tests/flutter/ios
INFO [2021-10-05 19:31:45.20]: -----------------------------------------------------
INFO [2021-10-05 19:31:45.20]: Step: flutter build ipa --flavor prod --dart-define\=STUDO_ENV\=PROD --export-options-plist\=ios/exportOptions.plist
INFO [2021-10-05 19:31:45.20]: -----------------------------------------------------
INFO [2021-10-05 19:31:45.20]: $ flutter build ipa --flavor prod --dart-define\=STUDO_ENV\=PROD --export-options-plist\=ios/exportOptions.plist
INFO [2021-10-05 19:31:46.01]: ▸ Changing current working directory to: /Users/runner/work/studo-tests/studo-tests/flutter
INFO [2021-10-05 19:31:51.28]: ▸ Archiving com.olmps.ticker...
INFO [2021-10-05 19:31:52.16]: ▸ Automatically signing iOS for device deployment using specified development team in Xcode project: 9TW867US97
INFO [2021-10-05 19:35:41.53]: ▸ Running pod install...                                            227.1s
INFO [2021-10-05 19:35:56.40]: ▸ Running Xcode build...
INFO [2021-10-05 19:35:56.40]: ▸ Xcode archive done.                                         14.8s
INFO [2021-10-05 19:36:00.04]: ▸ Failed to build iOS app
INFO [2021-10-05 19:36:00.05]: ▸ Error output from Xcode build:
INFO [2021-10-05 19:36:00.05]: ▸ ↳
INFO [2021-10-05 19:36:00.05]: ▸ ** ARCHIVE FAILED **
INFO [2021-10-05 19:36:00.05]: ▸ Xcode's output:
INFO [2021-10-05 19:36:00.05]: ▸ ↳
INFO [2021-10-05 19:36:00.05]: ▸ note: Using new build system
INFO [2021-10-05 19:36:00.05]: ▸ note: Building targets in parallel
INFO [2021-10-05 19:36:00.05]: ▸ note: Planning build
INFO [2021-10-05 19:36:00.05]: ▸ note: Constructing build description
INFO [2021-10-05 19:36:00.05]: ▸ warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.4.99. (in target 'AppAuth' from project 'Pods')
INFO [2021-10-05 19:36:00.05]: ▸ warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.4.99. (in target 'GoogleSignIn' from project 'Pods')
INFO [2021-10-05 19:36:00.05]: ▸ error: No signing certificate "iOS Distribution" found: No "iOS Distribution" signing certificate matching team ID "9TW867US97" with a private key was found. (in target 'Runner' from project 'Runner')
INFO [2021-10-05 19:36:00.05]: ▸ warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.4.99. (in target 'Flutter' from project 'Pods')
INFO [2021-10-05 19:36:00.06]: ▸ Encountered error while archiving for device.
WARN [2021-10-05 19:36:00.08]: Lane Context:
INFO [2021-10-05 19:36:00.08]: ***:DEFAULT_PLATFORM=>:ios, :PLATFORM_NAME=>:ios, :LANE_NAME=>"ios release", :KEYCHAIN_PATH=>"~/Library/Keychains/fastlane_tmp_keychain", :ORIGINAL_DEFAULT_KEYCHAIN=>"\"/Users/runner/Library/Keychains/login.keychain-db\"", :SIGH_PROFILE_TYPE=>"development", :MATCH_PROVISIONING_PROFILE_MAPPING=>***"com.olmps.ticker"=>"Studo Distribution", "com.olmps.ticker.dev"=>"Studo Development"***, :VERSION_NUMBER=>"0.1.0", :LATEST_BUILD_NUMBER=>21, :LATEST_VERSION=>"0.1.0", :LATEST_TESTFLIGHT_BUILD_NUMBER=>21, :LATEST_TESTFLIGHT_VERSION=>"0.1.0", :BUILD_NUMBER=>"22"***
ERROR [2021-10-05 19:36:00.08]: Exit status of command 'flutter build ipa --flavor prod --dart-define\=STUDO_ENV\=PROD --export-options-plist\=ios/exportOptions.plist' was 1 instead of 0.
Changing current working directory to: /Users/runner/work/studo-tests/studo-tests/flutter
Archiving com.olmps.ticker...
Automatically signing iOS for device deployment using specified development team in Xcode project: 9TW867US97
Running pod install...                                            227.1s
Running Xcode build...                                          
Xcode archive done.                                         14.8s
Failed to build iOS app
Error output from Xcode build:
↳
    ** ARCHIVE FAILED **

Xcode's output:
↳
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.4.99. (in target 'AppAuth' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.4.99. (in target 'GoogleSignIn' from project 'Pods')
    error: No signing certificate "iOS Distribution" found: No "iOS Distribution" signing certificate matching team ID "9TW867US97" with a private key was found. (in target 'Runner' from project 'Runner')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.4.99. (in target 'Flutter' from project 'Pods')

Encountered error while archiving for device.

INFO [2021-10-05 19:36:00.08]: Successfully generated documentation at path '/Users/runner/work/studo-tests/studo-tests/flutter/ios/fastlane/README.md'

+------+---------------------------------------------------------------+-------------+
|                                  fastlane summary                                  |
+------+---------------------------------------------------------------+-------------+
| Step | Action                                                        | Time (in s) |
+------+---------------------------------------------------------------+-------------+
| 1    | default_platform                                              | 0           |
| 2    | app_store_connect_api_key                                     | 0           |
| 3    | Switch to ios setup_provisions lane                           | 0           |
| 4    | setup_ci                                                      | 0           |
| 5    | match                                                         | 2           |
| 6    | match                                                         | 2           |
| 7    | flutter clean                                                 | 12          |
| 8    | flutter pub get                                               | 3           |
| 9    | get_version_number                                            | 0           |
| 10   | latest_testflight_build_number                                | 2           |
| 11   | increment_build_number                                        | 2           |
| 💥   | flutter build ipa --flavor prod --dart-define\=STUDO_ENV\=PRO | 254         |
+------+---------------------------------------------------------------+-------------+

ERROR [2021-10-05 19:36:00.19]: fastlane finished with errors

Looking for related GitHub issues on fastlane/fastlane...
Search query: Exit status of command 'flutter build ipa --flavor prod --dart-define\=STUDO_ENV\=PROD --export-options-plist\=ios/exportOptions.plist' was 1 instead of 0.
Changing current working directory to: /Users/runner/work/studo-tests/studo-tests/flutter
Archiving com.olmps.ticker...
Automatically signing iOS for device deployment using specified development team in Xcode project: 9TW867US97
Running pod install...                                            227.1s
Running Xcode build...                                          
Xcode archive done.                                         14.8s
Failed to build iOS app
Error output from Xcode build:
↳
    ** ARCHIVE FAILED **

Xcode's output:
↳
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.4.99. (in target 'AppAuth' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.4.99. (in target 'GoogleSignIn' from project 'Pods')
    error: No signing certificate "iOS Distribution" found: No "iOS Distribution" signing certificate matching team ID "9TW867US97" with a private key was found. (in target 'Runner' from project 'Runner')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.4.99. (in target 'Flutter' from project 'Pods')

Encountered error while archiving for device.

URL: https://api.github.com/search/issues?q=Exit%20status%20of%20command%20%27flutter%20build%20ipa%20--flavor%20prod%20--dart-define%5C%3DSTUDO_ENV%5C%3DPROD%20--export-options-plist%5C%3Dios%2FexportOptions.plist%27%20was%201%20instead%20of%200.%0AChanging%20current%20working%20directory%20to%3A%20%2FUsers%2Frunner%2Fwork%2Fstudo-tests%2Fstudo-tests%2Fflutter%0AArchiving%20com.olmps.ticker...%0AAutomatically%20signing%20iOS%20for%20device%20deployment%20using%20specified%20development%20team%20in%20Xcode%20project%3A%209TW867US97%0ARunning%20pod%20install...%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20227.1s%0ARunning%20Xcode%20build...%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%0AXcode%20archive%20done.%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%2014.8s%0AFailed%20to%20build%20iOS%20app%0AError%20output%20from%20Xcode%20build%3A%0A%E2%86%B3%0A%20%20%20%20%2A%2A%20ARCHIVE%20FAILED%20%2A%2A%0A%0A%0AXcode%27s%20output%3A%0A%E2%86%B3%0A%20%20%20%20note%3A%20Using%20new%20build%20system%0A%20%20%20%20note%3A%20Building%20targets%20in%20parallel%0A%20%20%20%20note%3A%20Planning%20build%0A%20%20%20%20note%3A%20Constructing%20build%20description%0A%20%20%20%20warning%3A%20The%20iOS%20deployment%20target%20%27IPHONEOS_DEPLOYMENT_TARGET%27%20is%20set%20to%208.0%2C%20but%20the%20range%20of%20supported%20deployment%20target%20versions%20is%209.0%20to%2014.4.99.%20%28in%20target%20%27AppAuth%27%20from%20project%20%27Pods%27%29%0A%20%20%20%20warning%3A%20The%20iOS%20deployment%20target%20%27IPHONEOS_DEPLOYMENT_TARGET%27%20is%20set%20to%208.0%2C%20but%20the%20range%20of%20supported%20deployment%20target%20versions%20is%209.0%20to%2014.4.99.%20%28in%20target%20%27GoogleSignIn%27%20from%20project%20%27Pods%27%29%0A%20%20%20%20error%3A%20No%20signing%20certificate%20%22iOS%20Distribution%22%20found%3A%20No%20%22iOS%20Distribution%22%20signing%20certificate%20matching%20team%20ID%20%229TW867US97%22%20with%20a%20private%20key%20was%20found.%20%28in%20target%20%27Runner%27%20from%20project%20%27Runner%27%29%0A%20%20%20%20warning%3A%20The%20iOS%20deployment%20target%20%27IPHONEOS_DEPLOYMENT_TARGET%27%20is%20set%20to%208.0%2C%20but%20the%20range%20of%20supported%20deployment%20target%20versions%20is%209.0%20to%2014.4.99.%20%28in%20target%20%27Flutter%27%20from%20project%20%27Pods%27%29%0A%0AEncountered%20error%20while%20archiving%20for%20device.%0A+repo:fastlane/fastlane
ERROR [2021-10-05 19:36:00.41]: Error finding relevant GitHub issues: undefined method `map' for nil:NilClass
/usr/local/lib/ruby/gems/2.7.0/gems/fastlane-2.195.0/fastlane_core/lib/fastlane_core/ui/interface.rb:153:in `shell_error!': \e[31m[!] Exit status of command 'flutter build ipa --flavor prod --dart-define\\=STUDO_ENV\\=PROD --export-options-plist\\=ios/exportOptions.plist' was 1 instead of 0. (FastlaneCore::Interface::FastlaneShellError)
Changing current working directory to: /Users/runner/work/studo-tests/studo-tests/flutter
Archiving com.olmps.ticker...
Automatically signing iOS for device deployment using specified development team in Xcode project: 9TW867US97
Running pod install...                                            227.1s
Running Xcode build...                                          
Xcode archive done.                                         14.8s
Failed to build iOS app
Error output from Xcode build:
↳
    ** ARCHIVE FAILED **

Xcode's output:
↳
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.4.99. (in target 'AppAuth' from project 'Pods')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.4.99. (in target 'GoogleSignIn' from project 'Pods')
    error: No signing certificate "iOS Distribution" found: No "iOS Distribution" signing certificate matching team ID "9TW867US97" with a private key was found. (in target 'Runner' from project 'Runner')
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.4.99. (in target 'Flutter' from project 'Pods')

Encountered error while archiving for device.
\e[0m
	from /usr/local/lib/ruby/gems/2.7.0/gems/fastlane-2.195.0/fastlane_core/lib/fastlane_core/ui/ui.rb:17:in `method_missing'
	from /usr/local/lib/ruby/gems/2.7.0/gems/fastlane-2.195.0/fastlane/lib/fastlane/helper/sh_helper.rb:80:in `sh_control_output'
	from /usr/local/lib/ruby/gems/2.7.0/gems/fastlane-2.195.0/fastlane/lib/fastlane/helper/sh_helper.rb:16:in `sh_no_action'
	from /usr/local/lib/ruby/gems/2.7.0/gems/fastlane-2.195.0/fastlane/lib/fastlane/fast_file.rb:223:in `block in sh'
	from /usr/local/lib/ruby/gems/2.7.0/gems/fastlane-2.195.0/fastlane/lib/fastlane/actions/actions_helper.rb:69:in `execute_action'
	from /usr/local/lib/ruby/gems/2.7.0/gems/fastlane-2.195.0/fastlane/lib/fastlane/fast_file.rb:222:in `sh'
	from /usr/local/lib/ruby/gems/2.7.0/gems/fastlane-2.195.0/fastlane/lib/fastlane/fast_file.rb:216:in `sh'
	from Fastfile:84:in `block (2 levels) in parsing_binding'
	from /usr/local/lib/ruby/gems/2.7.0/gems/fastlane-2.195.0/fastlane/lib/fastlane/lane.rb:33:in `call'
	from /usr/local/lib/ruby/gems/2.7.0/gems/fastlane-2.195.0/fastlane/lib/fastlane/runner.rb:49:in `block in execute'
	from /usr/local/lib/ruby/gems/2.7.0/gems/fastlane-2.195.0/fastlane/lib/fastlane/runner.rb:45:in `chdir'
	from /usr/local/lib/ruby/gems/2.7.0/gems/fastlane-2.195.0/fastlane/lib/fastlane/runner.rb:45:in `execute'
	from /usr/local/lib/ruby/gems/2.7.0/gems/fastlane-2.195.0/fastlane/lib/fastlane/lane_manager.rb:47:in `cruise_lane'
	from /usr/local/lib/ruby/gems/2.7.0/gems/fastlane-2.195.0/fastlane/lib/fastlane/command_line_handler.rb:36:in `handle'
	from /usr/local/lib/ruby/gems/2.7.0/gems/fastlane-2.195.0/fastlane/lib/fastlane/commands_generator.rb:109:in `block (2 levels) in run'
	from /usr/local/lib/ruby/gems/2.7.0/gems/commander-4.6.0/lib/commander/command.rb:187:in `call'
	from /usr/local/lib/ruby/gems/2.7.0/gems/commander-4.6.0/lib/commander/command.rb:157:in `run'
	from /usr/local/lib/ruby/gems/2.7.0/gems/commander-4.6.0/lib/commander/runner.rb:444:in `run_active_command'
	from /usr/local/lib/ruby/gems/2.7.0/gems/fastlane-2.195.0/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:117:in `run!'
	from /usr/local/lib/ruby/gems/2.7.0/gems/commander-4.6.0/lib/commander/delegates.rb:18:in `run!'
	from /usr/local/lib/ruby/gems/2.7.0/gems/fastlane-2.195.0/fastlane/lib/fastlane/commands_generator.rb:353:in `run'
	from /usr/local/lib/ruby/gems/2.7.0/gems/fastlane-2.195.0/fastlane/lib/fastlane/commands_generator.rb:42:in `start'
	from /usr/local/lib/ruby/gems/2.7.0/gems/fastlane-2.195.0/fastlane/lib/fastlane/cli_tools_distributor.rb:122:in `take_off'
	from /usr/local/lib/ruby/gems/2.7.0/gems/fastlane-2.195.0/bin/fastlane:23:in `<top (required)>'
	from /usr/local/lib/ruby/gems/2.7.0/bin/fastlane:25:in `load'
	from /usr/local/lib/ruby/gems/2.7.0/bin/fastlane:25:in `<main>'
Error: Process completed with exit code 1. </pre>
</details>

**Comment by ggirotto** [comment]
Env:

<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                                          |
| --------------------------- | -------------------------------------------------------------- |
| OS                          | 11.6                                                           |
| Ruby                        | 2.6.3                                                          |
| Bundler?                    | false                                                          |
| Git                         | git version 2.32.0                                             |
| Installation Source         | /usr/local/bin/fastlane                                        |
| Host                        | macOS 11.6 (20G165)                                            |
| Ruby Lib Dir                | /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib |
| OpenSSL Version             | LibreSSL 2.8.3                                                 |
| Is contained                | false                                                          |
| Is homebrew                 | false                                                          |
| Is installed via Fabric.app | false                                                          |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/                    |
| Xcode Version               | 13.0                                                           |
| Swift Version               | 5.5                                                            |


### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   | en_US.UTF-8 | ✅ |
| LANGUAGE |             |   |


### fastlane files:

<details><summary>`./fastlane/Fastfile`</summary>

```ruby
default_platform(:ios)

platform :ios do

  ########################--------------------------------########################
  ########################--------- SETUP LANES ----------########################
  ########################--------------------------------########################

  # Environment Setup
  ## Setup local environment configuration. If running on a CI machine we need to create a Keychain to store the 
  ## distribution certificates
  private_lane :setup_environment do
    if is_ci
      setup_ci
    end
  end

  # Provisions and Certificates Setup
  ## Download and setup all necessary provisions and certificates (Dev & Dist)
  desc "Setup Development & Distribution profiles and certificates in the local machine"
  lane :setup_provisions do
    setup_environment

    # Install Distribution Provisions and Certificates
    match(
      app_identifier: "com.olmps.ticker",
      type: "appstore",
      readonly: true,
      verbose: true,
    )

    # Install Development Provisions and Certificates
    match(
      app_identifier: "com.olmps.ticker.dev",
      type: "development",
      readonly: true,
      verbose: true,
    )
  end

  ########################--------------------------------########################
  ########################-------- RELEASE LANE ----------########################
  ########################--------------------------------########################

  # Release a new version to Testflight
  #
  # It assumes that the app version is correctly set.
  # The lane increments the build number if necessary
  desc "Release app to Testflight"
  lane :release do
    api_key = app_store_connect_api_key(
      key_id: ENV["APP_STORE_API_KEY_ID"],
      issuer_id: ENV["APP_STORE_API_KEY_ISSUER_ID"],
      key_content: ENV["APP_STORE_API_KEY"],
      in_house: false,
    )

    setup_provisions

    # Clean
    sh("flutter", "clean")

    # Pub get
    sh("flutter", "pub", "get")

    # Increment the build number (use the latest build number for this version + 1)
    increment_build_number(
      build_number: latest_testflight_build_number(
        initial_build_number: 0, 
        api_key: api_key, 
        version: get_version_number(configuration: "Release-prod")
      ) + 1,
      xcodeproj: "Runner.xcodeproj",
    )

    # Run flutter build with release mode
    sh("flutter", "build", "ipa", "--flavor", "prod", "--dart-define=STUDO_ENV=PROD", "--export-options-plist=ios/exportOptions.plist")
  end
end

```
</details>

<details><summary>`./fastlane/Appfile`</summary>

```ruby
app_identifier("com.olmps.ticker") # The bundle identifier of your app
apple_id(ENV["APPLE_ID"]) # Your Apple email address

itc_team_id(ENV["ITC_TEAM_ID"]) # App Store Connect Team ID
team_id(ENV["TEAM_ID"])
```
</details>

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.195.0 | ✅ Up-To-Date  |


### Loaded fastlane plugins:

**No plugins Loaded**


<details><summary><b>Loaded gems</b></summary>

| Gem                             | Version      |
| ------------------------------- | ------------ |
| did_you_mean                    | 1.5.0        |
| rouge                           | 2.0.7        |
| xcpretty                        | 0.3.0        |
| terminal-notifier               | 2.0.0        |
| unicode-display_width           | 1.7.0        |
| terminal-table                  | 1.8.0        |
| public_suffix                   | 4.0.6        |
| addressable                     | 2.8.0        |
| multipart-post                  | 2.0.0        |
| word_wrap                       | 1.0.0        |
| optparse                        | 0.1.1        |
| tty-screen                      | 0.8.1        |
| tty-cursor                      | 0.7.1        |
| tty-spinner                     | 0.9.3        |
| artifactory                     | 3.0.15       |
| babosa                          | 1.0.4        |
| colored                         | 1.2          |
| highline                        | 2.0.3        |
| commander                       | 4.6.0        |
| faraday-cookie_jar              | 0.0.7        |
| faraday_middleware              | 1.0.0        |
| gh_inspector                    | 1.1.3        |
| mini_magick                     | 4.11.0       |
| security                        | 0.1.3        |
| dotenv                          | 2.7.6        |
| simctl                          | 1.6.8        |
| retriable                       | 3.1.2        |
| memoist                         | 0.16.2       |
| multi_json                      | 1.15.0       |
| os                              | 1.1.1        |
| googleauth                      | 0.16.2       |
| httpclient                      | 2.8.3        |
| google-apis-core                | 0.4.0        |
| google-apis-androidpublisher_v3 | 0.8.0        |
| rexml                           | 3.2.5        |
| nanaimo                         | 0.3.0        |
| colored2                        | 3.1.2        |
| claide                          | 1.0.3        |
| CFPropertyList                  | 3.0.3        |
| atomos                          | 0.1.3        |
| xcodeproj                       | 1.20.0       |
| plist                           | 3.6.0        |
| excon                           | 0.83.0       |
| ruby2_keywords                  | 0.0.4        |
| faraday-patron                  | 1.0.0        |
| faraday-net_http_persistent     | 1.1.0        |
| faraday-net_http                | 1.0.1        |
| faraday-httpclient              | 1.0.1        |
| faraday-excon                   | 1.1.0        |
| faraday-em_synchrony            | 1.0.0        |
| faraday-em_http                 | 1.0.0        |
| faraday                         | 1.5.0        |
| unf_ext                         | 0.0.7.7      |
| unf                             | 0.1.4        |
| domain_name                     | 0.5.20190701 |
| http-cookie                     | 1.0.4        |
| fastimage                       | 2.2.4        |
| json                            | 2.5.1        |
| naturally                       | 2.2.1        |
| rubyzip                         | 2.3.1        |
| xcpretty-travis-formatter       | 1.0.1        |
| bundler                         | 2.2.9        |
| jwt                             | 2.2.3        |
| webrick                         | 1.7.0        |
| signet                          | 0.15.0       |
| mini_mime                       | 1.1.0        |
| trailblazer-option              | 0.1.1        |
| declarative                     | 0.0.20       |
| uber                            | 0.1.0        |
| representable                   | 3.1.1        |
| google-apis-playcustomapp_v1    | 0.5.0        |
| rake                            | 13.0.3       |
| digest-crc                      | 0.6.3        |
| google-apis-storage_v1          | 0.6.0        |
| google-apis-iamcredentials_v1   | 0.6.0        |
| google-cloud-errors             | 1.1.0        |
| google-cloud-env                | 1.5.0        |
| google-cloud-core               | 1.6.0        |
| google-cloud-storage            | 1.34.0       |
| emoji_regex                     | 3.2.2        |
| aws-eventstream                 | 1.1.1        |
| aws-sigv4                       | 1.2.3        |
| aws-partitions                  | 1.473.0      |
| jmespath                        | 1.4.0        |
| aws-sdk-core                    | 3.115.0      |
| aws-sdk-kms                     | 1.44.0       |
| aws-sdk-s3                      | 1.96.1       |
| forwardable                     | 1.3.2        |
| logger                          | 1.4.3        |
| date                            | 3.1.1        |
| stringio                        | 3.0.0        |
| ipaddr                          | 1.2.2        |
| openssl                         | 2.1.2        |
| zlib                            | 1.1.0        |
| mutex_m                         | 0.1.1        |
| ostruct                         | 0.3.3        |
| strscan                         | 3.0.0        |
| io-console                      | 0.5.8        |
| fileutils                       | 1.5.0        |
| libxml-ruby                     | 3.2.1        |
| psych                           | 3.3.0        |
</details>


*generated on:* **2021-10-05**
</details>

**Comment by ggirotto** [comment]
After a few days, I finally could find the solution. My problem was in the match repository certificates. I ended up redoing all the Match setup from the beginning and the error disappeared, because the distribution certificate was wrong.
